booster-app/src/main/java/de/neuefische/boosterapp/db/BoosterMongoDb.java

```diff
@@ -1,12 +1,15 @@
 package de.neuefische.boosterapp.db;
 
 import de.neuefische.boosterapp.model.Booster;
+import de.neuefische.boosterapp.model.BoosterType;
 import org.springframework.data.repository.PagingAndSortingRepository;
 
 import java.util.List;
 
 
 public interface BoosterMongoDb extends PagingAndSortingRepository<Booster,String>{
     List<Booster> findByCreator(String creator);
-    }
+    List<Booster> findByOwnerAndType(String owner, BoosterType type);
```

Comment:
with sample you can get a random doc so you dont need to load all documents for get a random booster  https://docs.mongodb.com/manual/reference/operator/aggregation/sample/